ansible/templates/ubuntu1604/deps.sh.j2

```diff
@@ -1,19 +1,37 @@
 {% if doc %}
-Install dependencies::
+Install dependencies:
+
+apt-get update
+
+apt-get -y install unzip
+apt-get -y install nginx
+
+apt-get -y install python3
+apt-get -y install python3-venv
+
 {% else %}
 #!/bin/bash
 
 set -e -u -x
 echo 'APT::Install-Recommends 0;' > /etc/apt/apt.conf.d/01norecommends \
  && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends
 
-{% endif %}
-
 apt-get update
-
 apt-get -y install unzip
+apt-get -y install nginx
+
+if [ "${PYTHONVER:-}" = "3.6" ]; then
+    apt-get install -y software-properties-common
+    add-apt-repository ppa:deadsnakes/ppa
```

Comment:
Was the intent to remove PPA usage everywhere?

Reply:
oops this one was forgotten
It should have been removed